client/src/main/java/org/spine3/client/ActorRequestFactory.java

```diff
@@ -245,21 +302,348 @@ public Query allWithMask(Class<? extends Message> entityClass, String... maskPat
+         *
+         * @return the built {@link Query}
+         */
+        public Query build() {
+            final FieldMask mask = composeMask();
+            // Implying AnyPacker.pack to be idempotent
+            final Set<Any> entityIds = composeIdPredicate();
+
+            final Query result = composeQuery(targetType, entityIds, columns, mask);
+            return result;
+        }
+
+        @Nullable
+        private FieldMask composeMask() {
+            if (fieldMask == null || fieldMask.isEmpty()) {
+                return null;
+            }
+            final FieldMask mask = FieldMask.newBuilder()
+                                            .addAllPaths(fieldMask)
+                                            .build();
+            return mask;
+        }
+
+        @Nullable
+        private Set<Any> composeIdPredicate() {
+            if (ids == null || ids.isEmpty()) {
+                return null;
+            }
+            final Collection<Any> entityIds = transform(ids, new Function<Object, Any>() {
+                @Nullable
+                @Override
+                public Any apply(@Nullable Object o) {
+                    checkNotNull(o);
+                    final Any id = Identifiers.idToAny(o);
+                    return id;
+                }
+            });
+            final Set<Any> result = newHashSet(entityIds);
+            return result;
+        }
+
+        @SuppressWarnings("MethodWithMoreThanThreeNegations")
+            // OK for this method as it's used primarily for debugging
+        @Override
+        public String toString() {
+            final String valueSeparator = "; ";
+            final StringBuilder sb = new StringBuilder();
+            sb.append(QueryBuilder.class.getSimpleName())
+              .append('(')
+              .append("SELECT ");
+            if (fieldMask == null || fieldMask.isEmpty()) {
+                sb.append('*');
+            } else {
+                sb.append(fieldMask);
+            }
+            sb.append(" FROM ")
+              .append(targetType.getSimpleName())
+              .append(" WHERE (");
+            if (ids != null && !ids.isEmpty()) {
+                sb.append("id IN ")
+                  .append(ids)
+                  .append(valueSeparator);
+            }
+            if (columns != null && !columns.isEmpty()) {
+                for (Map.Entry<String, Any> column : columns.entrySet()) {
+                    sb.append(column.getKey())
+                      .append('=')
+                      .append(Json.toCompactJson(unpack(column.getValue())))
+                      .append(valueSeparator);
+                }
+            }
+            sb.append(");");
+            return sb.toString();
+        }
+    }
+
+    /**
+     * A parameter of a {@link Query} to the read side.
+     *
+     * <p>This class may be considered as a filter for the query. An instance contains the name of
```

Comment:
Please read [this](https://www.englishforums.com/English/ToBeConsideredAs/bbcmjr/post.htm), as that's a common mistake in the JavaDocs you create.